crates/ibc/src/clients/ics07_tendermint/header.rs

```diff
@@ -249,7 +249,7 @@ mod pretty {
     }
 }
 
-#[cfg(any(test, feature = "mocks"))]
+#[cfg(any(test, feature = "test-utils"))]
 pub mod test_util {
```

Comment:
Should we use `test_utils` as convention? I see we have `crate::test_utils` now.

```suggestion
pub mod test_utils {
```

Reply:
The codebase under `test_utils` is now under `ibc-testkit`, so these modules are removed.